src/components/MainContent/Components.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Element } from 'react-scroll/modules';
+
+export interface ContentProps {
+  children:
+    | void
+    | string
+    | JSX.Element
+    | JSX.Element[]
+    | (JSX.Element | undefined)[]
+    | (JSX.Element | undefined);
+}
+
+export interface SectionProps extends ContentProps {
+  name: string;
+}
+
+export const Section = ({ name, children }: SectionProps) => (
+  <Element name={name}>
+    {children}
+    <style jsx>{`
+      padding-left: 0.5em;
```

Comment:
This isn't valid CSS, you still need to have a selector with curly brackets. I know that means wrapping the component in another `div` or `section`, but I think it's for the best.